usr/share/rear/lib/help-workflow.sh

```diff
@@ -23,17 +23,18 @@ $PRODUCT comes with ABSOLUTELY NO WARRANTY; for details see
 the GNU General Public License at: http://www.gnu.org/licenses/gpl.html
 
 Available options:
- -h --help           usage information (this text)
- -c DIR              alternative config directory; instead of $CONFIG_DIR
- -C CONFIG           additional config files; absolute path or relative to config directory
- -d                  debug mode; run many commands verbosely with debug messages in log file (also sets -v)
- -D                  debugscript mode; log executed commands via 'set -x' (also sets -v and -d)
- --debugscripts SET  same as -d -v -D but debugscript mode with 'set -SET'
- -r KERNEL           kernel version to use; currently '$KERNEL_VERSION'
- -s                  simulation mode; show what scripts are run (without executing them)
- -S                  step-by-step mode; acknowledge each script individually
- -v                  verbose mode; show messages what $PRODUCT is doing on the terminal or show verbose help
- -V --version        version information
+ -h --help              usage information (this text)
+ -c DIR                 alternative config directory; instead of $CONFIG_DIR
+ -C CONFIG              additional config files; absolute path or relative to config directory
+ -d                     debug mode; run many commands verbosely with debug messages in log file (also sets -v)
+ -D                     debugscript mode; log executed commands via 'set -x' (also sets -v and -d)
+ --debugscripts SET     same as -d -v -D but debugscript mode with 'set -SET'
+ -r KERNEL              kernel version to use; currently '$KERNEL_VERSION'
+ -s                     simulation mode; show what scripts are run (without executing them)
+ -S                     step-by-step mode; acknowledge each script individually
+ -v                     verbose mode; show messages what $PRODUCT is doing on the terminal or show verbose help
+ -V --version           version information
+ -n --non-interactive   non-interactive mode; aborts when any user input is required
```

Comment:
Same as above:
I think non-interactive mode does not abort
when **any** user input is required,
but only when user input is required via the UserInput function
under certain timeout conditions.